# Patch to avoid ETXBUSY when trying to replace mamake.exe
--- a/bin/package	Thu Aug 01 18:54:10 2024
+++ b/bin/package	Mon Mar 10 20:42:45 2025
@@ -2464,7 +2464,7 @@
 				# To allow relinking the binary against libast later, add a _bootstrap suffix and create
 				# a symlink to it. The symlink can be removed/renamed instead of the _bootstrap binary.
				$exec $CC -O $CCFLAGS $LDFLAGS -o "$INSTALLROOT/bin/${i}_bootstrap" "$INITROOT/$i.c" || return
-				$exec ln -sf "${i}_bootstrap" "$INSTALLROOT/bin/$i" || return
+				MSYS="$MSYS winsymlinks:sys" $exec ln -sf "${i}_bootstrap" "$INSTALLROOT/bin/$i" || return
				;;
 			*)	$exec $CC -O $CCFLAGS $LDFLAGS -o "$INSTALLROOT/bin/$i" "$INITROOT/$i.c" || return
 				;;
